lib/changelogerator.rb

```diff
@@ -65,6 +58,17 @@ def self.changes_files_in_paths?(change, paths)
     nil
   end
 
+  # Return the label code for a change
+  # if the label name matches the expected pattern.
+  # nil otherwise.
+  def self.get_label_code(name)
+    if match = name.match(/^([a-z])(\d+)-(.*)$/i)
+      letter, number, text = match.captures
+      return [letter, number, text]
+    end
+    nil
+  end
```

Comment:
I feel like this could be more 'rubyish' (rubicop complains about the assignment in the if statement, for example), but can't think of anything nicer right now.